src/data-display/dynamic/dynamic-chart/templates/treemap/index.vue

```diff
@@ -125,7 +125,8 @@ export default defineComponent<DynamicChartTreemapProps>({
 
             const chart = am4core.create(ctx, TreeMap);
 
-            drawTreemapChart(chart, props.nameOptions, props.valueOptions);
+            const totalValue = sum(state.filteredData.map(d => d.value));
```

Comment:
This component doesn't know what key the value is in.
The key path information is in `valueOptions` prop.
Also, must defend against the case where the value is `undefined`.

```suggestion
            const totalValue = sum(state.filteredData.map(d => d[props.valueOptions.key] ?? 0));
```

Reply:
Thanks, I changed the code like that.